crates/sui-benchmark/src/benchmark/transaction_creator.rs

```diff
@@ -154,10 +156,10 @@ impl TransactionCreator {
         use_move: bool,
         chunk_size: usize,
         num_chunks: usize,
-        sender: Option<&KeyPair>,
+        sender: Option<&AccountKeyPair>,
         validator_preparer: &mut ValidatorPreparer,
     ) -> Vec<(Transaction, CertifiedTransaction)> {
-        let (address, keypair) = if let Some(a) = sender {
+        let (address, keypair): (_, AccountKeyPair) = if let Some(a) = sender {
```

Comment:
Wondering if for readability we should replace `_` and point to the real Address type here? Same comment applies in every place where we use `(_, Account/AuthorityKeyPair)`.

Wondering if we should actually have two functions for key generation, one for accounts and another for authorities. Then we could avoid always adding the expected type in every invocation.

Reply:
I think we don't need to, because the thing that is at _ is not generic. I personally think it makes the code more verbose that way.